### ISSUE TITLE
[Usable- Bot Framework Web Chat - Card Wizard]: 'Upload file' button is not consistent while navigating after selecting white labelled experience.

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5368]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:

Users with cognitive limitations and users with low vision, will get impacted as that they will not be able to interact properly or will not be able to understand and navigate properly. 

### Note: 
User credentials should NOT be included in the bug.
 

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Suggested action, Others or unrelated

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type Card Wizard word in type your message text box.
4. Navigate to the search results. (select copilot, fluent ui experience and type Card Wizard)
5. Verify the issue.

### What do you expect?

'Upload file' button should be consistent while navigating after selecting white labelled experience.

### What actually happened?

'Upload file' button is not consistent while navigating after selecting white labelled experience.
### Observation:
After selecting 'Fluent ui, copilot' experience the 'upload file' button is after the 'type your message' edit box.
In 'white labelled' experience the 'upload file' button before the 'type your message' edit box.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/9090f28a-cd53-4faf-8dc9-8fdd21f2086d)

**Comment by manojs5** [comment]
#A11ySeV4;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#A11yUsable;#Win11-Edge(Chromium);

**Comment by OEvgeny** [comment]
Different Web Chat variants are expected to have different layout. In real world scenarios there likely won't be such switch control.
